efficientdet/keras/train_lib.py

```diff
@@ -174,7 +289,7 @@ def get_optimizer(params):
   if params['optimizer'].lower() == 'sgd':
     logging.info('Use SGD optimizer')
     optimizer = tf.keras.optimizers.SGD(
-        learning_rate, momentum=params['momentum'])
+        learning_rate, momentum=params['momentum'], nesterov=True)
```

Comment:
do you get better results with nesterov?

Reply:
I removed nesterov and got 39 mAP0.5